### PR TITLE
prefix suffix can be node

### PR DESCRIPTION
**Pull request by hwigmore**
for instance, when entering a password in Spacer Pro, after the min number of characters we show a checkmark. 

added a react preview example
screen shot 2018-09-14 at 3 12 38 pm

@emma-floored @Jinsoo94

### CODE REVIEW

File: react/input/TextField.jsx [hunk 1]
```diff
@@ -96,10 +96,13 @@ TextField.propTypes = {
...
   label: PropTypes.string,
   placeholder: PropTypes.string,
-  prefix: PropTypes.string,
+  prefix: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.node
+  ]),
   suffix: PropTypes.oneOfType([
     PropTypes.string,
-    PropTypes.object
```
Comment (hwigmore) [inline react/input/TextField.jsx]: do you know if `PropTypes.object` would ever be needed / work how we have this set up? i think not?

Comment (emma-floored) [reply react/input/TextField.jsx]: oh I think I didn't know `.node` was a type and I used `.object` to also mean that

**Comment by hwigmore** [comment]
ok to merge?

**Comment by emma-floored** [comment]
👍

**Pull request merged**